src/main/java/figure/AbstractFigure.java

```diff
@@ -0,0 +1,9 @@
+package figure;
+
+public abstract class AbstractFigure {
+    public abstract void paint();
+
```

Comment:
let's add field color here, because that repeats in few classes

Reply:
added

Reply:
added